StudioTBA/CoronaIO
# Game scene with tile creation

## What issue is this PR solving?
Closes: #21 

## Description of change
Given a floor scale and a tile prefab, cover the floor by placing tiles on top. This is needed for the procedural generation of the map.

## How to reproduce the changes
1. Go to scene _Game_
2. Select game object _Floor_
3. In the inspector, go to _Tile Generator_ script and adjust the floor scale to your liking. **Note:** the scale should be a multiple of 10 for optimal results
4. Start the scene
5. See the new floor
6. Bonus: Inspect Assets/Scripts/TileGenerator to see how this is done

## Comments
The reason the scale of the tile should be 5 and that of the floor should be a multiple of 10 is so all the tiles fit perfectly on the floor. In other words, for uniformity. A floor of scale 50x50 (x, z) and a tile of scale 5x5 (x, z) would result in 10 rows and 10 tiles per row. If we were to have a tile of scale 4x4 for the same floor, then this would result in 12.5 rows and 12.5 tiles per row, which could create problems when procedurally generating the map and object placement.
